### PR TITLE
Generate QR codes for public keys #103

### PR DESCRIPTION
**Pull request by bshevchenko**
Снимок экрана 2019-07-31 в 1 33 52

Only one TODO: update QR code button icon image that is requested in issue
Identicon is not there since actual screen height doesn't equal design screen height

### CODE REVIEW

File: src/ts/components/account/QR.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  }
+}
+
+const mapStateToProps = (state: IAppState) => {
+  return {
+    settings: state.settings
+  }
+}
+
+const mapDispatchToProps = { saveSettings, setAccounts }
```
Comment (mytechtip) [inline src/ts/components/account/QR.tsx]: Are these used? if not, can be removed.

File: src/ts/components/account/QR.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+type DispatchProps = typeof mapDispatchToProps
+
+type StateProps = ReturnType<typeof mapStateToProps>
+
+const PublicKey = connect(mapStateToProps)(styled.div`
```
Comment (mytechtip) [inline src/ts/components/account/QR.tsx]: Do we need `connect(mapStateToProps)` here?

**Comment by bshevchenko** [comment]
@mytechtip what exactly you'd like to be covered with tests?

**Comment by mytechtip** [comment]
> @mytechtip what exactly you'd like to be covered with tests?

Good question! We can’t find much to be unit tested. So forget about it.

**Pull request merged**